clang/tools/clang-scan-deps/ClangScanDeps.cpp

```diff
@@ -230,6 +234,11 @@ static llvm::cl::opt<ResourceDirRecipeKind> ResourceDirRecipe(
     llvm::cl::init(RDRK_ModifyCompilerPath),
     llvm::cl::cat(DependencyScannerCategory));
 
+llvm::cl::opt<bool> EmitCASCompDB(
+    "emit-cas-compdb",
+    llvm::cl::desc("Emit compilation DB via CAS based dependency scanning."),
```

Comment:
I think this help message doesn't quite describe what it is doing. Maybe something like:
"Emit compilation DB with updated clang arguments for CAS based dependency scanning build"? Maybe you can phrase it better than that but the main point is to update clang arguments, right?

Reply:
Good point! I'm going to rephrase to make it clearer.

Reply:
Done.